bin/build.mjs

```diff
@@ -55,16 +55,46 @@ async function buildPackage(absWorkingDir) {
    * @param {'esm' | 'cjs'} format
    */
   const entryPoints = (format) => {
-    let outFilename = filename + (format === 'esm' ? '.module' : '');
-    outFilename += DEV ? '' : '.min';
+    /**
+     * @param {string} base
+     */
+    const addSuffix = (base) => {
+      let outFilename = base + (format === 'esm' ? '.module' : '');
+      return outFilename + (DEV ? '' : '.min');
+    };
+    let outFilename = addSuffix(filename);
+
+    const extraEntryPoints = ['@pixiv/three-vrm', '@pixiv/three-vrm-materials-mtoon'].includes(packageJson.name)
+      ? { [addSuffix('nodes/index')]: 'src/nodes/index.ts' }
+      : {};
+
     return {
       entryPoints: {
         [outFilename]: 'src/index.ts',
```

Comment:
I think we can directly put `addSuffix(filename)` here as `extraEntryPoints` does

Reply:
[e2e96c9](https://github.com/pixiv/three-vrm/pull/1427/commits/e2e96c9a27992cf7001ba81e27350643ae97dc7f)